### PR TITLE
snake-ize event names too

### PR DESCRIPTION
**Pull request by jmileham**
/domain @samandmoore @noveng05

### CODE REVIEW

File: app/models/test_track/analytics_event.rb [hunk 1]
```diff
@@ -10,9 +10,9 @@ def initialize(assignment)
 
     def name
       if assignment.feature_gate?
-        'FeatureGateExperienced'
+        'feature_gate_experienced'
       else
-        'SplitAssigned'
+        'split_assigned'
```
Comment (noveng05) [inline app/models/test_track/analytics_event.rb]: do these mixpanel [specs](https://github.com/Betterment/test_track_rails_client/blob/master/spec/models/test_track/analytics/mixpanel_client_spec.rb#L12) need some updating too?

Also, just curious whether this "breaks" the continuity of some funnels being tracked in mixpanel?  Do folks need to update what they're tracking to look at either `SplitAssigned` or `split_assigned` when this lands in the apps?

**Comment by noveng05** [comment]
<< domaintafn
just a couple of questions to better my understanding

**Comment by jmileham** [comment]
Yeah, so the context is I failed to snake-ize the event names when i snake-ized the property names. `split_experienced` is the event name we already use these days, just making TestTrack generate that name by default rather than have to override it elsewhere. I'll release alpha3 and then upgrade to use this once it lands.

**Comment by noveng05** [comment]
got it. thanks for the context. 
domainlgtm

File: app/models/test_track/analytics_event.rb (hunk 1 above)
Comment (jmileham) [reply app/models/test_track/analytics_event.rb]: I can update these specs too though they're not actually hurting anything, just a little less realistic.

**Comment by jmileham** [comment]
This passes locally gonna merge optimistically.

**Pull request merged**